tests/dragonfly/shutdown_test.py

```diff
@@ -9,10 +9,13 @@
 BASIC_ARGS = {"dir": "{DRAGONFLY_TMP}/"}
 
 
+@pytest.mark.skip(reason='Currently we can not guarantee that on shutdown if command is executed and value is written we response before breaking the connction')
```

Comment:
there is a typo `connction`

Reply:
@romange fixed this. Please take a look on the last commit message. I will use it in the squashing